Microsoft.Toolkit.Uwp/Microsoft.Toolkit.Uwp.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Toolkit\Microsoft.Toolkit.csproj" />
+    <PackageReference Include="CommunityToolkit.Common" Version="7.1.2" />
```

Comment:
This is a massively breaking change for all consumers.
I thought we had agreed to keep referencing the `Microsoft.Toolkit` package for now?
Even if we wanted to change this, it should be done in a separate PR.
cc. @michael-hawker

Reply:
Yet, if I leave the project reference as is, you'll hit a problem come compile time when it searches for a project that no longer exists on disk.

Reply:
No I get that, but what I'm saying is, can't we still switch to the `PackageReference` (since the project is gone now, as you said), but reference `Microsoft.Toolkit` instead? I mean, that's on NuGet as well, is what I'm saying 🙂

Reply:
We may still want to ship a new hotfix if the XMLNS stuff works for UWP/WinUI. So let's move to Microsoft.Toolkit that's on NuGet for now and then we can update this to use CommunityToolkit.Common when we ship an 8.0 later.